search-resources-api/src/test/java/no/unit/nva/search/SearchAllHandlerTest.java

```diff
@@ -3,7 +3,9 @@
 import static java.net.HttpURLConnection.HTTP_OK;
 import static no.unit.nva.search.RequestUtil.DOMAIN_NAME;
 import static no.unit.nva.search.RequestUtil.PATH;
-import static no.unit.nva.search.SearchAllHandler.PAGE_SIZE_QUERY_PARAM;
+import static no.unit.nva.search.SearchAllHandler.DEFAULT_RESULTS_SIZE;
+import static no.unit.nva.search.SearchAllHandler.PAGE_QUERY_PARAM;
+import static no.unit.nva.search.SearchAllHandler.RESULTS_QUERY_PARAM;
 import static no.unit.nva.search.SearchHandler.DEFAULT_PAGE_SIZE;
 import static no.unit.nva.search.SearchHandler.EXPECTED_ACCESS_RIGHT_FOR_VIEWING_MESSAGES_AND_DOI_REQUESTS;
 import static no.unit.nva.search.SearchHandler.VIEWING_SCOPE_QUERY_PARAMETER;
```

Comment:
we have a repetition of constants here. Better to move the constants in its own class